uaclient/tests/test_apt.py

```diff
@@ -69,7 +69,7 @@ def test_write_apt_pin_file_to_apt_preferences(self, m_platform, tmpdir):
         expected_pref = dedent(
             """\
             Package: *
-            Pin: release o=MYORIG, n=xenial
```

Comment:
I bet we can drop the [mocked get_platform_info](https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/tests/test_apt.py#L58) this related test now.